lib/testing-helpcenter/data/page_title_list.rb

```diff
@@ -0,0 +1,12 @@
+# frozen_string_literal: true
+
+require 'json'
+# Open JSON file and parse it's data to PageTitleList object
+class PageTitleList
+  attr_reader :hash
+
+  def initialize(file_path)
+    @raw_data = File.read(file_path)
```

Comment:
Right now there is no point to making `@raw_data` class variable, so `raw_data = ..` is better than `@raw_data = ...`